### PR TITLE
CI container builds

### PR DESCRIPTION
**Pull request by aasseman**

### CODE REVIEW

**Comment by aasseman** [comment]
A container image got uploaded for this PR: https://github.com/semiotic-ai/timeline-aggregation-protocol/pkgs/container/tap_aggregator

File: .github/workflows/containers.yml
Comment (gusinacio) [inline .github/workflows/containers.yml:11]: Do you want to build for every PR to branch main?

**Review by gusinacio — APPROVED** [review]
LGTM, just want to confirm if it's the intended use

File: .github/workflows/containers.yml
Comment (aasseman) [reply .github/workflows/containers.yml:11]: Yeah, I was thinking that it could make integration testing easier, if we want to do that before committing to merge new PRs.

**Pull request merged**
